data/autoyast_sle15/autoyast_bcache.xml

```diff
@@ -0,0 +1,218 @@
+      <partitions config:type="list">
+        <partition>
+          <bcache_caching_for config:type="list">
+            <listentry>/dev/bcache0</listentry>
+          </bcache_caching_for>
+          <create config:type="boolean">false</create>
+          <format config:type="boolean">false</format>
+          <resize config:type="boolean">false</resize>
+        </partition>
+      </partitions>
+    </drive>
+    <drive>
+      <type config:type="symbol">CT_BCACHE</type>
+      <device>/dev/bcache0</device>
+      <bcache_options>
+        <cache_mode>writethrough</cache_mode>
+      </bcache_options>
+      <use>all</use>
+      <partitions config:type="list">
+        <partition>
+          <mount>/home</mount>
+          <size>max</size>
+        </partition>
+        <partition>
+          <mount>swap</mount>
+          <filesystem config:type="symbol">swap</filesystem>
+          <size>auto</size>
+        </partition>
+      </partitions>
+    </drive>
+  </partitioning>
+  <software>
+    <packages config:type="list">
+      <package>glibc</package>
+      <package>grub2</package>
+      <package>snapper</package>
+      <package>syslinux</package>
+      <package>kdump</package>
+      <package>sles-release</package>
+      <package>kexec-tools</package>
+    </packages>
+    <patterns config:type="list">
+      <pattern>apparmor</pattern>
+      <pattern>base</pattern>
+      <pattern>basesystem</pattern>
+      <pattern>enhanced_base</pattern>
+      <pattern>minimal_base</pattern>
+      <pattern>yast2_basis</pattern>
+    </patterns>
+    <products config:type="list">
+      <product>SLES</product>
+    </products>
+  </software>
+  <report>
+    <errors>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">0</timeout>
+    </errors>
+    <messages>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">0</timeout>
+    </messages>
+    <warnings>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">0</timeout>
+    </warnings>
+    <yesno_messages>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">0</timeout>
+    </yesno_messages>
+  </report>
+  <networking>
+    <interfaces config:type="list">
+      <interface>
+        <bootproto>dhcp</bootproto>
+        <device>eth0</device>
+        <dhclient_set_default_route>yes</dhclient_set_default_route>
+        <startmode>auto</startmode>
+      </interface>
+    </interfaces>
+  </networking>
+  <firewall>
+    <enable_firewall config:type="boolean">true</enable_firewall>
+    <start_firewall config:type="boolean">true</start_firewall>
+  </firewall>
+  <timezone>
+    <hwclock>UTC</hwclock>
+    <timezone>Europe/Berlin</timezone>
+  </timezone>
+  <users config:type="list">
```

Comment:
Check 'users' as well, sometimes is not needed as far as I remember, if you just need to use root for the validation. If so, in the validation below we can use another path not relative to `bernhard`.

Reply:
i would rather leave it as it is as it doesnt make any difference.